google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -274,6 +409,7 @@ private Builder(SessionPoolOptions options) {
       this.actionOnSessionNotFound = options.actionOnSessionNotFound;
       this.actionOnSessionLeak = options.actionOnSessionLeak;
       this.trackStackTraceOfSessionCheckout = options.trackStackTraceOfSessionCheckout;
+      this.actionOnInactiveTransaction = options.actionOnInactiveTransaction;
```

Comment:
This is the method called from public `toBuilder` method. We are not setting `inactiveTransactionRemovalOptions` here since its only used in our tests and hence is not needed to be passed through public methods. Customers only need to set `actionOnInactiveTransaction` by calling one of the two public method `setCloseIfInactiveTransactions` or `setWarnIfInactiveTransactions`.

Reply:
There are many other fields as well that are not set by users, but we still set them in this constructor. Imagine the confusion if someone adds a test that actually uses this field, and then calls this method to create a copy of their SessionPoolOptions and the value for this field not being copied (while everything else is).